routes/web.php

```diff
@@ -38,11 +40,12 @@
     }
 });
 
-Statamic::additionalWebRoutes();
+Route::middleware(Statamic::webMiddleware())->group(function () {
+    Statamic::additionalWebRoutes();
```

Comment:
Did you have a particular reason to wrap this in the middleware stack too?

(Excuse the delay)

Reply:
No particular reason, but i thought that additional routes should also have these middlewares (Localize & Cache).